internal/linkhelpers/linkhelpers.go

```diff
@@ -2,21 +2,40 @@ package linkhelpers
 
 import (
 	"bufio"
+	"context"
+	"database/sql"
 	"errors"
 	"fmt"
+	_ "github.com/jackc/pgx/v5/stdlib"
 	"github.com/size12/url-shortener/internal/config"
 	"net/url"
 	"os"
 	"sync"
+	"time"
 )
 
+var Error409 = errors.New("link is already in storage")
```

Comment:
```suggestion
var Err409 = errors.New("link is already in storage")
```
префикс для таких ошибок Err

Reply:
поменял, приму к сведению